### PR TITLE
Publish myst-lldb. Add tests/lldb

### PR DESCRIPTION
**Pull request by anakrish**
The test for myst-lldb is basic and performs exactly the
same checsk that gdb test does: ensure that a break point is hit,
and that we are able to print the backtrace and continue execution
until program termination.

Signed-off-by: Anand Krishnamoorthi <anakrish@microsoft.com>

### CODE REVIEW

File: .azure_pipelines/ci-pipeline-makefile-nightly.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ jobs:
         sudo apt-get install build-essential python3-setuptools python3-pip llvm-7 libmbedtls-dev docker-ce -y
         sudo chmod 666 /var/run/docker.sock
         sudo apt install python3-pip -y
+	sudo apt install lldb-10 -y
```
Comment (salsal97) [inline .azure_pipelines/ci-pipeline-makefile-nightly.yml:35]: yaml is space-sensitive, it seems to be failing out because it needs one more tab here

Comment (anakrish) [reply .azure_pipelines/ci-pipeline-makefile-nightly.yml]: Thanks! trying it out now.

File: tests/lldb/Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+include $(TOP)/defs.mak
+
+APPDIR = appdir
+CFLAGS = -fPIC -g
+LDFLAGS = -Wl,-rpath=$(MUSL_LIB)
+LLDB_LOG_FILE = myst_lldb_out.log
+
+LLDB_CMDS=-o "b print_hello" \
+	-o "run" \
+        -o "process handle SIGILL -s false -n false" \
```
Comment (mingweishih) [inline tests/lldb/Makefile:11]: nit: alignment

**Review by vtikoo — APPROVED** [review]
Looks good!

**Review by mingweishih — APPROVED** [review]

**Pull request merged**